src/operator/activation-inl.h

```diff
@@ -24,7 +24,7 @@ namespace op {
 namespace activation {
 enum ActivationOpInputs {kData};
 enum ActivationOpOutputs {kOut};
-enum ActivationOpType {kReLU, kSigmoid, kTanh, kSoftReLU};
+enum ActivationOpType {kReLU, kSigmoid, kTanh, kSoftReLU, kLinear};
```

Comment:
Thanks. I missed that you can just leave out the activation. I used Keras before, where a dense layers always has an activation. Removing this.